app/models/cart.rb

```diff
@@ -18,6 +18,10 @@ def count_total
     data.values.sum
   end
 
+  def total_price
+    (data.map { |k, v| Item.find(k.to_i).price * v }).reduce(:+)
+  end
+
```

Comment:
We now have a `total_price` method in both the Cart and Order model. Are they doing the same thing? If so we should pull them out.